app/src/pages/prompt/PromptLayout.tsx

```diff
@@ -93,8 +95,11 @@ export function PromptLayout() {
         index={tabIndex}
         onChange={(index) => {
           let url: string;
+          alert(index);
           if (index === 1) {
             url = `/prompts/${loaderData.prompt.id}/versions`;
+          } else if (index === 2) {
+            url = `/prompts/${loaderData.prompt.id}/config`;
           } else {
             url = `/prompts/${loaderData.prompt.id}`;
           }
```

Comment:
Sorry about this terrible system 😂

Reply:
tab index is such a strange paradigm. Hopefully aria components make this better and more declarative